### ISSUE TITLE
bug: Module not found: Can't resolve 'deepmerge'

### ISSUE THREAD

**Issue by kyonenya** [mustaphaturhan/chakra-ui-markdown-renderer#14]
## Reproduction Procedure

I got this error when I simply create app with `crate-next-app` and install packages using `npm`, and set `ChakraProvider` to `app.js`.
`pages/index.js` is as follows.

```tsx
import ChakraUIRenderer from 'chakra-ui-markdown-renderer';
import ReactMarkdown from 'react-markdown';

export default function Home() {
  const markdown = `## Heading
  paragraph`;

  return (
    <ReactMarkdown
      components={ChakraUIRenderer()}
      children={markdown}
    />
  )
}
```

Then I got this error.

```
Failed to compile
./node_modules/chakra-ui-markdown-renderer/dist/index.esm.js:1:258
Module not found: Can't resolve 'deepmerge'
null
```

The package version is 3.0.0 (latest).

```
"chakra-ui-markdown-renderer": "^3.0.0",
```

I have made a repo for this issue.

Gitpod: 
https://peach-stoat-lwzsdygc.ws-us04.gitpod.io/
GitHub Repository:
https://github.com/kyonenya/deepmerge-chakra-ui-markdown-renderer

## Cause

Package `deepmerge` is used but not explitly installed in `package.json`.
So, when I install manually `deepmerge 4.2.2`, then it works correctly.

In addition, `"@types/deepmerge": "^2.2.0"` is in `devDependencies`,
but this packages seems to be deprecated.

> deepmerge provides its own type definitions, so you don't need @types/deepmerge installed!
> https://www.npmjs.com/package/@types/deepmerge

**Comment by mustaphaturhan** [comment]
Hey @kyonenya, thank you for this awesome issue. I don't know how but I wasn't aware that I forgot to add deepmerge to dependencies. I am working on it, will release a bug fix today.

**Comment by mustaphaturhan** [comment]
Fixed with v3.0.1.

**Comment by kyonenya** [comment]
What a quick fix! Thanks a lot.
